src/Engine/ProtoCore/Lang/BuiltInMethods.cs

```diff
@@ -130,6 +135,8 @@ public enum MethodID
             Constants.kNodeAstFailed,   // kNodeAstFailed
             "__GC",                     // kGC
             Constants.kIfConditionalMethodName,
+            "__ToStringFromObjectAndFormat",     // kToStringFromObjectAndFormat
+            "__ToStringFromArrayAndFormat",      // kToStringFromArrayAndFormat
```

Comment:
can't figure out why we need `__ToStringFromArrayAndFormat`. Can't replication work with `__ToStringFromObjectAndFormat` ? I must be missing something..

Reply:
yes replication works, but we need these overrides when you want a single string output, not a bunch of strings.